### PR TITLE
Add Microsoft.Framework.Runtime.Internals.xproj

### PR DESCRIPTION
**Pull request by troydai**
The xproj file is missing.

### CODE REVIEW

File: src/Microsoft.Framework.Runtime.Internals/Microsoft.Framework.Runtime.Internals.xproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="__ToolsVersion__" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (dougbu) [inline src/Microsoft.Framework.Runtime.Internals/Microsoft.Framework.Runtime.Internals.xproj]: Ick. Run one of the scripts to fix this file up e.g. \\GarnetA\share\UpdateXProjectFiles\UpdateXProjectFiles.ps1

File: DNX.sln [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+VisualStudioVersion = 14.0.22815.1
```
Comment (dougbu) [inline DNX.sln]: Ah. So time can run backwards.

File: src/Microsoft.Framework.Runtime.Internals/Microsoft.Framework.Runtime.Internals.xproj [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>1bbfbd55-a127-47a1-a510-a3d3cd7c6232</ProjectGuid>
```
Comment (dougbu) [inline src/Microsoft.Framework.Runtime.Internals/Microsoft.Framework.Runtime.Internals.xproj]: Could avoid touching the .sln file by pasting `693A73B9-621A-4184-B90C-9B9FF67E8CE8` in here.

**Comment by dougbu** [comment]
:watch:

**Comment by troydai** [comment]
ping @dougbu

**Comment by davidfowl** [comment]
Is it also missing in the release branch?

**Comment by troydai** [comment]
yeah, will update release branch and merge to dev

**Comment by troydai** [comment]
I'll close this one and open a new one based on release branch.

**Comment by troydai** [comment]
https://github.com/aspnet/dnx/pull/2000

**Pull request closed without merging**